.coveragerc

```
@@ -249,7 +249,7 @@ omit =
     homeassistant/components/ecowitt/sensor.py
     homeassistant/components/eddystone_temperature/sensor.py
     homeassistant/components/edimax/switch.py
-    homeassistant/components/edl21/*
+    homeassistant/components/edl21/sensor.py
```

I think init is fully untested for now.
I am not sure why coverage doesn't pick it up...
```suggestion
    homeassistant/components/edl21/__init__.py
    homeassistant/components/edl21/sensor.py
```